content/en/install.md

```diff
@@ -3,101 +3,206 @@ title: Installation
 
-You can install SciPy from PyPI with `pip`:
+[install-uv]: https://docs.astral.sh/uv/getting-started/installation/
 
-    python -m pip install scipy
+2.  Create a new project in a new subdirectory, by executing the following in a terminal:
 
-<a name="conda-install"></a>
+        uv init try-scipy
+        cd try-scipy
 
-# Installing via Conda
+    {{< admonition hint >}}
+The second command changes directory into the directory of your project.
+    {{< /admonition >}}
 
-You can install SciPy from the `defaults` or `conda-forge` channels with `conda`:
+3.  Install Python:
 
-    conda install scipy
+        uv python install
 
-<a name="package_manager"></a>
+    {{< admonition note >}}
+If you have already installed Python, this step is optional.
+    {{< /admonition >}}
 
-# Install system-wide via a package manager
+4.  Add SciPy to your project:
+
+        uv add scipy
+
+5.  Try out SciPy!
+
+        uv run python
+
+    This will launch a Python interpreter session, from which you can `import scipy`.
+
+See next steps in [the SciPy user guide][scipy-user-guide].
+
+[scipy-user-guide]: https://docs.scipy.org/doc/scipy/tutorial/
+
+### Installing with `pixi`
+
+If you work with non-Python packages, you may prefer to install SciPy as
+a [Conda] package, so that you can use the same workflow for packages which
+are not available on [PyPI](https://pypi.org/), the Python Package Index.
+Conda can manage packages in any language, so you can use it to install
+Python itself, compilers, and other languages.
+
+[Conda]: https://docs.conda.io/projects/conda/en/latest/index.html
+
+The steps to install SciPy from [conda-forge] using the package management
+tool [`pixi`] are very similar to the steps for `uv`:
+
+[conda-forge]: https://conda-forge.org/
+[`pixi`]: https://pixi.sh/latest/
+
+1. Install `pixi`, following [the instructions in the `pixi` documentation][install-pixi].
+
+[install-pixi]: https://pixi.sh/latest/
+
+2.  Create a new project in a new subdirectory:
+
+        pixi init try-scipy
+        cd try-scipy
+
+3.  Add SciPy to your project:
+
+        pixi add scipy
+
+    {{< admonition note >}}
+This step also adds Python from conda-forge.
+    {{< /admonition >}}
+
+4.  Try out SciPy!
+
+        pixi run python
+
+See next steps in [the SciPy user guide][scipy-user-guide].
+
+<a name="environment-based"></a>
+
+## Environment-based workflows
+
+In environment-based workflows, you install packages into an environment, which you
+can activate and deactivate.
+These workflows are well-established,
+but lack some reproducibility benefits of project-based workflows.
+
+### Installing with `pip`
+
+1.  [Install Python](https://www.python.org/downloads/).
+
+2.  Create and activate a virtual environment with `venv`.
+
+   {{< admonition hint >}}
+See [the tutorial in the Python Packaging User Guide](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments).
+   {{< /admonition >}}
```

Comment:
this indentation looks redundant - the admonition doesn't indent as there is no body of the bullet to bind to, it seems.